bftclient/include/bftclient/bft_client.h

```diff
@@ -26,6 +26,7 @@
 #include "metrics.h"
 #include "diagnostics.h"
 #include "bftengine/Crypto.hpp"
+#include "seq_num_generator.h"
```

Comment:
Just FYI: I specifically left this out of the bft_client.h file because it should be user defined. There are many ways to generate sequence numbers. This is something Guy and I discussed at length with Simple Client. There are comments in the code specifying different ways.

Reply:
fixed